app/models/profile.rb

```diff
@@ -159,10 +159,6 @@ def activate_after_fraud_review_unnecessary
   def activate_after_passing_in_person
     transaction do
       update!(
-        fraud_review_pending_at: nil,
-        fraud_rejection_at: nil,
-        fraud_pending_reason: nil,
-        deactivation_reason: nil,
         in_person_verification_pending_at: nil,
       )
       activate
```

Comment:
👍 

It took me a minute to reconcile this with the PR body, which says:

> This PR also improves the `Profile.activate_after_passing_in_person` method by not allowing profiles with deactivation reasons or fraud pending reasons to be activated.

This occurs because the call to `activate` checks if there is a `deactivation_reason`, and it is no longer being unconditionally set to nil here.

Reply:
Ah yeah good call out I will add this to the description. We noticed this issue with some of our testing around password reset causing an encryption error. It was notice in the following scenario: Before the user logged in after password reset, the Profiles were becoming active after passing proofing then profiles would change back to being deactivated with deactivation reason `encryption_error` after the user logged in without a personal key.

Reply:
Oh, sorry, my comment was vague; I didn't mean to create any work for you. I was just leaving a note for other reviewers.

Password reset + IPP was a little bit of a minefield when I was on the team. I'm glad to see the user experience is improving. 🎉